app/Models/User.php

```diff
@@ -59,8 +59,14 @@ class User extends Authenticatable
      */
     protected $appends = [
         'profile_photo_url',
+        'can',
```

Comment:
by appending the dynamic attribute, we ensure its going to be always available in the frontend user object